tools/concat_multi_datasets/catWrapper.xml

```diff
@@ -331,13 +331,23 @@
 
 **WARNING:** The paired collection operations do not handle gziped files.
 
+**WARNING:** When concatenating 2 collections make sure the first collection is the one with the most items.
+
 -----
 
 **What it does**
 
 Concatenates datasets and paired collections with multiple options:
 
- - It's possible select either a concatenation by strand, by pair or a whole collection concatenation, when the input is a paired collection.
+ - When the input is a paired collection:
+
+   - concatenation by strand : all the forward files are concatenated then all the reverse files are concatenated to finish with a single forward - reverse pair
+
+   - concatenation by pair : all the pairs are concatenated to finish with a collection containing single files
```

Comment:
forward - reverse dataset pairs are concatenated and a simple dataset collection is returned